yolov5/val.py

```diff
@@ -497,7 +496,7 @@ def run(
                                 # Merge the instance into the session (updates if already exists)
                                 session.merge(image_processing_status)
                     else:
-                        LOGGER.debug('Area to blur is 0.')
+                        LOGGER.warning('Area to blur is 0.')
```

Comment:
Can you rebase? this code is removed now. We just ignore the not existing bounding boxes. It produces perf overhead